docs/README.md

```diff
@@ -183,6 +185,20 @@ To ensure that your client side localization files are valid, run:
 sail npm run i18n:validate
 ```
 
+## Job Queues
+
+As the Mismatch Finder includes jobs to validate and import mismatch CSV files, it is possible to also configure the job queue runner to run locally.
+
+By Default, the jobs will run synchronously to the requests that dispatch them. However, in order to queue and run jobs from the database, simple follow these two steps:
+
+1. In your local `.env` file, change the value of `QUEUE_CONNECTION` to `database` (it should be set to `sync` by default).
+
+1. Start the job queue runner by typing in the following command: 
+
+    ```
+    sail artisan queue:listen --timeout=1200
```

Comment:
Maybe it would be more complete if you indicate briefly why `1200` is a good number to put there?

Reply:
Good point, thank you!